src/main/java/com/amazon/janusgraph/graph/AbstractGraphFactory.java

```diff
@@ -0,0 +1,10 @@
+package com.amazon.janusgraph.graph;
+
+import org.janusgraph.graphdb.database.StandardJanusGraph;
+
+/**
+ * Created by addisonslabaugh on 6/22/17.
+ */
+public interface AbstractGraphFactory {
```

Comment:
Succinct is better - probably can be done in one or two lines in an all in one test method.